webapi/_test_data/all_data/beol-data.ttl

```diff
@@ -1459,3 +1459,40 @@
 	knora-base:hasSubListNode <http://rdfh.ch/lists/0801/youth> , <http://rdfh.ch/lists/0801/personal_contact> , <http://rdfh.ch/lists/0801/correspondence> , <http://rdfh.ch/lists/0801/books> , <http://rdfh.ch/lists/0801/living_circumstances> , <http://rdfh.ch/lists/0801/miscellanea> ;
 	rdfs:label "Personal life"@en .
 
+<http://rdfh.ch/lists/0801/optics> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Optics" ;
+    knora-base:listNodePosition 0 ;
+    knora-base:hasSubListNode <http://rdfh.ch/lists/0801/theoryOfLight>;
+    rdfs:label "Optics"@en .
+
+<http://rdfh.ch/lists/0801/theoryOfLight> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Theory of Light" ;
+    knora-base:listNodePosition 0 ;
+    knora-base:hasSubListNode <http://rdfh.ch/lists/0801/theoryOfReflection> , <http://rdfh.ch/lists/0801/theoryOfColors> , <http://rdfh.ch/lists/0801/glassExperiment>;
+    rdfs:label "Theory of Light"@en .
+
+<http://rdfh.ch/lists/0801/theoryOfReflection> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Theory of Reflection" ;
+    knora-base:listNodePosition 0 ;
+    rdfs:label "Theory of Reflection"@en .
+
+<http://rdfh.ch/lists/0801/theoryOfColors> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Theory of Colors" ;
+    knora-base:listNodePosition 0 ;
+    rdfs:label "Theory of Colors"@en .
+
+<http://rdfh.ch/lists/0801/glassExperiment> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Newton's Experiment of Glass" ;
+    knora-base:listNodePosition 0 ;
+    rdfs:label "Newton's Experiment of Glass"@en .
+
+<http://rdfh.ch/lists/0801/royalSocietyElection> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Royal Society Election" ;
+    knora-base:listNodePosition 0 ;
+    rdfs:label "Royal Society Election"@en .
```

Comment:
Could you add a newline to the end of the file?

Reply:
done!

Reply:
done!